src/SprykerEco/Zed/AfterPay/Persistence/AfterPayQueryContainerInterface.php

```diff
@@ -17,6 +17,9 @@
 interface AfterPayQueryContainerInterface extends QueryContainerInterface
 {
     /**
+     * Specification:
+     * - TODO: Add method specification.
+     *
```

Comment:
if you don't know what to write here - revert these changes and commit with `-n` flag

Reply:
This is actually the new sniffer rule. So it must be commited like this anyway, better yet of course with some actual text.